src/components/FilterTabs/index.tsx

```diff
@@ -0,0 +1,75 @@
+import { FC } from "react"
+import { Button, ButtonGroup, useStyleConfig } from "@chakra-ui/react"
+import { FilterTabSize, FilterTabVariant } from "../../theme/FilterTab"
+
+export interface FilterTabProps {
+  title: string
+  tabId: string
+  onClick: (tabId: string) => void
+  selectedTabId: string
+  variant: FilterTabVariant
+  size: FilterTabSize
+}
+
+export interface FilterTabsProps {
+  variant?: FilterTabVariant
+  size?: FilterTabSize
+  selectedTabId: string
+  setSelectedTabId: (tabId: string) => void
+  tabs: {
+    title: string
+    tabId: string
+    onClick?: (tabId: string) => void
+  }[]
+}
+
+const FilterTab: FC<FilterTabProps> = ({
+  selectedTabId,
+  tabId,
+  onClick,
+  title,
+  variant,
+  size,
+}) => {
+  const isActive = selectedTabId === tabId
```

Comment:
I would create a context in `FilterTab` and use context here to get the selected tab id.